packages/disploy/src/structs/BaseChannel.ts

```diff
@@ -18,9 +18,12 @@ export abstract class BaseChannel extends Base {
 	 * The type of the channel.
 	 */
 	public abstract type: ChannelType;
-
+	/**
+	 * The raw data of the channel.
+	 */
+	declare public raw: APIChannel;
 	public constructor(app: App, raw: APIChannel) {
-		super(app);
+		super(app, raw);
```

Comment:
While this would probably be a good idea to let the user access the raw objects; it is out of scope for this PR.